docs/getting-started/installation/frc-plugin.md

```diff
@@ -0,0 +1,6 @@
+# Installing the FRC plugin for IntelliJ
+To get things like the right build tasks, class templates, and project templates in IntelliJ you need the FRC plugin. To install this you need to open IntelliJ, and navigate to `File > Settings > Plugins` and then click on the "Marketplace" tab. You're going to want to search "FRC" and download the plugin by Mark Vedder (also shows as Javaru for some people).
```

Comment:
> To get things

It's good practice to avoid vague words like 'things' and 'stuff' and the like. 

I also think this should be a section within the intellij install guide, since they go hand in hand

Also, a picture is worth 1,000 words. Add a screenshot of the plugins page